apps/sms/views/conversation/js/conversation.js

```diff
@@ -2248,18 +2244,25 @@ var ConversationView = {
     this.container.classList.remove('hide');
   },
 
-  // FIXME/bug 983411: phoneNumber not needed.
-  simSelectedCallback: function conv_simSelected(phoneNumber, cardIndex) {
```

Comment:
Ha-ha, yeah, but you forgot about changes for multisim action button here :) Anyway it's better to fix it in a dedicated bug and keep PR as small as possible (not like I usually do :P - I'm working on this :))